pkg/util/kubernetes/clustername/clustername.go

```diff
@@ -120,8 +120,11 @@ func getClusterName(ctx context.Context, data *clusterNameData, hostname string)
 				}
 			}
 			if len(clusterName) > 0 {
-				log.Infof("Using cluster name %s from the node label", clusterName)
-				data.clusterName = clusterName
+				if !IsRFC1123CompliantClusterName(clusterName) {
+					log.Infof("Cluster name \"%s\" is not RFC 1123 compliant, it will be converted, ", clusterName)
```

Comment:
should this be a warning instead of info?

Reply:
yes that a good idea

Reply:
```suggestion
					log.Warnf("Cluster name \"%s\" is not RFC 1123 compliant, it will be converted, ", clusterName)
```